YACReader/main_window_viewer.cpp

```diff
@@ -1596,6 +1476,46 @@ void MainWindowViewer::getSiblingComics(QString path, QString currentComic)
     updatePrevNextActions(index > 0, index + 1 < list.count());
 }
 
+void MainWindowViewer::setActionsEnabled(bool enabled)
+{
+    //TODO enable goTo and showInfo (or update) when numPages emited
+    const auto actions = { saveImageAction,
+                           goToPageOnTheLeftAction,
+                           goToPageOnTheRightAction,
+                           adjustHeightAction,
+                           adjustWidthAction,
+                           goToPageAction,
+                           leftRotationAction,
+                           rightRotationAction,
+                           showMagnifyingGlassAction,
+                           doublePageAction,
+                           doubleMangaPageAction,
+                           adjustToFullSizeAction,
+                           fitToPageAction,
+                           showZoomSliderlAction,
+                           increasePageZoomAction,
+                           decreasePageZoomAction,
+                           resetZoomAction,
+                           showBookmarksAction,
+                           showInfoAction,
+                           showDictionaryAction,
+                           showFlowAction };
+    for (auto *a : actions)
+        a->setEnabled(enabled);
+}
+
+void MainWindowViewer::setMglassActionsEnabled(bool enabled)
+{
+    for (auto *a : qAsConst(mglassActions))
+        a->setEnabled(enabled);
+}
+
+void MainWindowViewer::setLoadedComicActionsEnabled(bool enabled)
+{
+    for (auto *a : qAsConst(loadedComicActions))
```

Comment:
[`qAsConst`](https://doc.qt.io/qt-5/qtglobal.html#qAsConst) used here and in `MainWindowViewer::setMglassActionsEnabled()` was introduced in Qt 5.7, so YACReader will require Qt 5.7+ when this PR is merged. It would be better to use standard [`std::as_const`](https://en.cppreference.com/w/cpp/utility/as_const) when we start requiring C++17.

Reply:
No longer a problem because Qt 5.15 is the minimum supported version now. Switched to `std::as_const` as C++17 is now required.